src/Compilers/CSharp/Portable/Errors/XmlSyntaxDiagnosticInfo.cs

```diff
@@ -38,22 +33,6 @@ protected override DiagnosticInfo GetInstanceWithSeverityCore(DiagnosticSeverity
             return new XmlSyntaxDiagnosticInfo(this, severity);
         }
 
-        #region Serialization
-
-        protected override void WriteTo(ObjectWriter writer)
-        {
-            base.WriteTo(writer);
-            writer.WriteUInt32((uint)_xmlErrorCode);
-        }
-
-        private XmlSyntaxDiagnosticInfo(ObjectReader reader)
-            : base(reader)
-        {
-            _xmlErrorCode = (XmlParseErrorCode)reader.ReadUInt32();
-        }
-
-        #endregion
-
         public override string GetMessage(IFormatProvider? formatProvider = null)
         {
             var culture = formatProvider as CultureInfo;
```

Comment:
writing this comment here above the generated file.  THe generated file is where the buld of the deletion comes from as we no longer need to emit all that goop for syntax serialization.